src/main/java/com/exactpro/th2/fix/client/fixBean/BaseFixBean.java

```diff
@@ -75,34 +72,33 @@ public void setHeartBtInt(long heartBtInt) {
     }
 
     public void setValidateUserDefinedFields(String validateUserDefinedFields) {
-        if (!YES_OR_NO.matcher(validateUserDefinedFields).matches()) {
+        if (!validateUserDefinedFields.equals("Y") && !validateUserDefinedFields.equals("N")) {
```

Comment:
> !validateUserDefinedFields.equals("Y") && !validateUserDefinedFields.equals("N")

Check like this can be extracted into a method (maybe even include throwing an exception into it to make it look similar to `Objects.requireNonNull`). Please also check if lowercase values are valid for QFJ

Reply:
lowercase values doesn't valid.